devops/arangodb.yml

```diff
@@ -9,6 +10,10 @@
     - name: arangodb_root_password
       prompt: "Password for ArangoDB root account"
 
+    - name: ssl_cert_path
+      prompt: "Path to parent ssl cert directory"
```

Comment:
Maybe this answers the question above about how the user gets the SSL cert into the Vagrant setup.

I don't think this needs to be a prompt though. The password is prompted for in order to provide an option for not echoing the password itself to the screen. This option can just be passed in via command line argument.

Reply:
Vagrant doesn't prompt for these vars, which is why I had to specify the password in the Vagrantfile. It does take the defaults if they exist though. If a user wanted to not use the default path, they would have to add another argument to `raw_arguments` to the Vagrantfile